Gems/DiffuseProbeGrid/Code/Source/EditorComponents/EditorDiffuseGlobalIlluminationComponent.cpp

```diff
@@ -27,7 +27,7 @@ namespace AZ
                     editContext->Class<EditorDiffuseGlobalIlluminationComponent>(
                         "Diffuse Global Illumination", "Diffuse Global Illumination configuration")
                         ->ClassElement(Edit::ClassElements::EditorData, "")
-                        ->Attribute(Edit::Attributes::Category, "Atom")
+                        ->Attribute(Edit::Attributes::Category, "Graphics - Light")
```

Comment:
You have 'Light' here instead of 'Lights'

Reply:
Changed it to Lighting too, per your suggestion.